src/alfasim_sdk/_internal/constants.py

```diff
@@ -396,11 +396,29 @@ class ControllerType(Enum):
 
 
 class PigTrappingMode(Enum):
+    """
+    Trapping mode of a PIG equipment.
+
+    - Automatic: the Pig is automatically trapped whenever it reaches a boundary node (e.g.,
```

Comment:
We can actually document each enum value using a special comment above it:

```python
class PigTrappingMode(Enum):
    """
    Trapping mode of a PIG equipment.    
    """

    #: The Pig is automatically trapped whenever it reaches a boundary node (e.g.,
    #: pressure or mass boundary).
    Automatic = "automatic"

    #: The Pig is trapped according to user-defined pipe and position unless it
    #: reaches a boundary node first.
    UserDefined = "user_defined"
```

This works for every type of class definition, not being specific to enum.

This renders it like this:

```
![image](https://user-images.githubusercontent.com/1085180/139660737-1e935124-3d1d-4202-8398-db6313be0a9f.png)
```

Reply:
I changed those locally to test them, decided to just push the changes. 👍

Reply:
That is nice, didn't know about that.